### PR TITLE
Small clean up for Sage

### PR DESCRIPTION
**Pull request by Log1x**
## Change log

- Use `app.globals` for enabling global helpers instead of a filter    
- Set default timezone using `app.timezone`
- Set default mb encoding to `UTF-8`
- Add docblocks to Sage class
- Clean up Sage class, move to anonymous functions
- Add default comments template (`partials/comments.blade.php`)
- Move Search Form filter from Sage to Acorn
- Move Body Class filter from Sage to Acorn
- **Add Composer autoloading** (#29)

Sage has already been updated to include `app.timezone` and `app.globals`. If you're down with moving the filters over, I'll remove them from Sage as well as remove the specific defining `/partials/comments.blade.php` in partials such as `content-single.blade.php`.

### CODE REVIEW

**Comment by QWp6t** [comment]
I don't want to use anonymous functions because that prohibits theme authors from removing those filters.

File: src/Acorn/Sage/Sage.php [hunk 1]
```diff
@@ -13,53 +13,153 @@
...
     public function attachCommentsTemplateResolver($priority = 100)
     {
-        add_filter('comments_template', [$this, 'commentsTemplateResolver'], $priority);
+        add_filter('comments_template', function ($name) {
+            if (Str::startsWith($file, [STYLESHEETPATH, TEMPLATEPATH])) {
+                $file = ltrim(str_replace([STYLESHEETPATH, TEMPLATEPATH], '', $file), '\\/');
+            }
+
+            if ($file == '/comments.php') {
+                $file = '/partials/comments.blade.php';
```
Comment (QWp6t) [inline src/Acorn/Sage/Sage.php]: This should be `'partials/comments.blade.php'` to be consistent with `ltrim()` above and because it's a relative path.

**Comment by Log1x** [comment]
All of the filters have been separated into their own functions as well as cleaned up, documented, etc.

I have also added View Composer autoloading which subsequently solves #29 and will remove a step when it comes to working with Sage, etc.

**Comment by QWp6t** [comment]
I removed the following changes:
* Set default timezone using app.timezone: We can discuss in separate issue/pr if you want, but I think Bedrock would be the more appropriate place for this.
* Set default mb encoding to UTF-8: [WordPress already handles this](https://github.com/WordPress/WordPress/blob/master/wp-settings.php#L377) [programmatically](https://developer.wordpress.org/reference/functions/wp_set_internal_encoding/).

**Pull request merged**